Update performance config docs for Nethermind (#895)
smartcontractkit/documentation: docs/Node Operators/evm-performance-configuration.md

RPCTxFeeCap = 0.0
```

> > 🚧 If you are using `Nethermind` instead of `go-ethereum` you should be aware that the `gascap` flag behaves differently, resulting in errors like [this one](https://github.com/NethermindEth/nethermind/issues/4307).
> > In this case, just set `gascap` to a high number like `5000000000` or just don't set it at all, to avoid issues.

Reviewer: ```suggestion
> 🚧 If you are using `Nethermind` instead of `go-ethereum` you should be aware that the `gascap` flag behaves differently, resulting in errors like [this one](https://github.com/NethermindEth/nethermind/issues/4307).
> In this case, you can set `gascap` to a very large number like `5000000000`.
```
